tinytodo/src/policy_store.rs

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2022-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use std::{
+    path::{Path, PathBuf},
+    time::Duration,
+};
+
+use notify::{
+    event::{AccessKind, AccessMode, RemoveKind},
+    Config, Error, Event, EventKind, RecommendedWatcher, Watcher,
+};
+use tokio::sync::mpsc::Sender;
+use tracing::{error, info, trace};
+
+use crate::context::{AppQuery, AppQueryKind};
+
+#[tracing::instrument]
+fn watch_function(result: std::result::Result<Event, Error>) {
+    trace!("Got notify event");
+    match result {
+        Ok(ev) => match ev.kind {
+            EventKind::Access(AccessKind::Close(AccessMode::Write)) => todo!(),
+            EventKind::Remove(RemoveKind::File) => {
+                info!("Policy set file updated!")
```

Comment:
But this doesn't seem to be doing anything?

Reply:
Ack you're right. Had some git versioning issues and think I pushed the wrong version.